src/table/Table.js

```diff
@@ -0,0 +1,255 @@
+import { Table as FixedDataTable } from 'fixed-data-table';
+
+import './patch-fixed-data-table';
+
+import 'fixed-data-table/dist/fixed-data-table-base.min.css';
+import './table.scss';
+
+const ARROW_UP = 38;
+const ARROW_DOWN = 40;
+
+export const isInvariantSatisfied = ({ selectionType, onRowsSelect, onRowSelect }) => {
+  const multipleSelectionEnabled = selectionType === 'multi';
+  /* eslint-disable no-console, quotes */
+  if (onRowSelect && onRowsSelect) {
+    console.warn("'onRowSelect' and 'onRowsSelect' are exclusive. Use the former if 'multipleSelectionEnabled' is true, or the latter if it's false");
+    return false;
+  } else if (multipleSelectionEnabled && onRowSelect) {
+    console.warn("'multipleSelectionEnabled' is true, so 'onRowSelect' will never be called. Use 'onRowsSelect' (plural) instead.");
+    return false;
+  } else if (!multipleSelectionEnabled && onRowsSelect) {
+    console.warn("'multipleSelectionEnabled' is false, so 'onRowsSelect' will never be called. Use 'onRowSelect' (singular) instead.");
+    return false;
+  }
+  return true;
+  /* eslint-enable no-console, quotes */
+};
+
+const Props = t.subtype(t.struct({
+  /** the desired height of the table */
+  height: t.maybe(t.Number),
+  /** the desired width of the table */
+  width: t.maybe(t.Number),
+  /** if true, the table will fit the container if width and height */
+  autoSize: t.maybe(t.Boolean),
+  /** if true, the columns will split evenly the width, avoiding horizontal scrollbars */
+  autoSizeColumns: t.maybe(t.Boolean),
+  /** height in pixel of every row */
+  rowHeight: t.maybe(t.Number),
+  /** height in pixel of header */
+  headerHeight: t.maybe(t.Number),
+  /** height in pixel of groupHeader */
+  groupHeaderHeight: t.maybe(t.Number),
+  /** height in pixel of footer */
+  footerHeight: t.maybe(t.Number),
+  /** method to get the desired row of data */
+  rowGetter: t.Function,
+  /** number of row displayed in the table */
+  rowsCount: t.maybe(t.Number),
+  cellRenderer: t.Function,
+  children: t.ReactNode,
+  sort: t.maybe(t.struct({
+    by: t.maybe(t.String),
+    dir: t.maybe(t.enums.of(['asc', 'desc'], 'sortDir'))
+  })),
+
+  selectedRows: t.maybe(t.list(t.Num)),
+  onRowSelect: t.maybe(t.Func),
+  onRowsSelect: t.maybe(t.Func),
+  selectionType: t.enums.of(['none', 'single', 'multi'], 'selectionType'),
+
+  className: t.maybe(t.String),
+  style: t.maybe(t.Object),
+  id: t.maybe(t.String)
+}), isInvariantSatisfied, 'FixedDataTableWrapperProps');
+
+@pure
+@skinnable()
+@props(Props)
+export default class Table extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      height: this.props.height,
+      width: this.props.width,
+      scrollToRow: this.props.selectedRows[0]
+    };
+  }
+
+  static defaultProps = {
+    width: 400,
+    height: 400,
+    selectedRows: []
+  };
+
+  componentDidMount() {
+    this.node = React.findDOMNode(this.refs.wrapper);
```

Comment:
REMINDER: I changed this (and maybe other littles things) in https://github.omnilab.our.buildo.io/buildo/labonline/pull/1499